index.md

```diff
@@ -1,15 +1,49 @@
 ---
 layout: default
-title: Secret Hitler Strategy
+title: The Unofficial Secret Hitler Strategy Guide
 ---
 
-# Secret Hitler Strategy
+# The Unofficial Secret Hitler Strategy Guide
 
-Strategies learned from 300+ games
+[Secret Hitler](http://secrethitler.com/) is a social deduction game for 5–10 people about finding and stopping the Secret Hitler.
 
-### Contributors
+The majority of players are liberals. If they can learn to trust each other, they have enough votes to control the table and win the game. But some players are fascists. They will say whatever it takes to get elected, enact their agenda, and blame others for the fallout. The liberals must work together to discover the truth before the fascists install their cold-blooded leader and win the game.
 
-Simon Brand, Fraser Cormack, Sean Jones, Mark M. Miller, Meenakshi Ravindran, Tatjana Starcenko
+This is an unofficial strategy guide for Secret Hitler which attempts to immortalise the insight gained from our group's 300+ games in the form of a living document. We will continue to update this site as our metagame is continuously butchered and put back together into some horrifying meta-Frankenstien.
```

Comment:
This is an unofficial strategy guide for Secret Hitler. We wrote it to immortalise the insight gained from our group's 400+ games in the form of a living document. We will continue to update this guide as we butcher our metagame and reassemble it into some horrifying meta-Frankenstein's monster.